pkg/apis/trust/v1alpha1/types_bundle.go

```diff
@@ -168,24 +168,34 @@ type BundleStatus struct {
 // BundleCondition contains condition information for a Bundle.
 type BundleCondition struct {
 	// Type of the condition, known values are (`Synced`).
-	Type BundleConditionType `json:"type"`
+	// The regex it matches is (dns1123SubdomainFmt/)?(qualifiedNameFmt)
```

Comment:
Where can I go to understand an example of `(dns1123SubdomainFmt/)?(qualifiedNameFmt)`? What is the qualified name format? Could we maybe rephrase this to use examples rather than a psuedo-regex-y thing?

Reply:
Good point! I just copied the text from the upstream Condition. 😊 Maybe adding these regex'es is something we should not do - even if it's a deviation from the upstream API?

Reply:
I think it's fine to add the regex - but I agree with James that the upstream text that's been copied here is not super clear. Maybe we just remove this line - anyone who cares what the regex matches can copy it into some regex matcher site. Most probably, nobody will care because it'll only be maintainers who set bundle conditions!

Reply:
Removed it now.